taichi/codegen/codegen_llvm.h

```diff
@@ -219,11 +219,12 @@ class CodeGenLLVM : public IRVisitor, public LLVMModuleBuilder {
 
   void visit(SNodeOpStmt *stmt) override;
 
-  llvm::Value *atomic_add_quant_fixed(AtomicOpStmt *stmt, CustomFloatType *cft);
+  llvm::Value *atomic_add_quant_fixed(AtomicOpStmt *stmt,
+                                      CustomFixedType *cfxt);
 
   llvm::Value *atomic_add_quant_int(AtomicOpStmt *stmt, CustomIntType *cit);
 
-  llvm::Value *quant_fixed_to_quant_int(CustomFloatType *cft,
+  llvm::Value *quant_fixed_to_quant_int(CustomFixedType *cfxt,
```

Comment:
nit: Here and below, can these args be made `const`?

Reply:
Yep. I'll fix it along with my next PR.